src/app/extensions/quickorder/shared/quickorder-csv-form/quickorder-csv-form.component.html

```diff
@@ -25,6 +25,13 @@ <h3>{{ 'quickorder.page.csv.title' | translate }}</h3>
           >
           <small *ngSwitchDefault></small>
         </div>
+        <p>
+          {{ 'quickorder.page.csv.sample.before' | translate }}
+          <a href="assets/quickorder-sample/quickorder.csv" download rel="nofollow">
+            {{ 'quickorder.page.csv.sample.link' | translate }}
+          </a>
+          {{ 'quickorder.page.csv.sample.after' | translate }}
```

Comment:
This needs to be handled with only a single translation key. Splitted sentences are a no go.
See https://github.com/intershop/intershop-pwa/blob/develop/src/assets/i18n/en_US.json#L144 for a working example.